libs/langchain-community/src/utils/bedrock/index.ts

```diff
@@ -117,6 +117,12 @@ export interface BaseBedrockInput {
   */
   model: string;
 
+  /** Optional URL Encoded model alias to use. Necessary for invoking an Application Inference Profile
+      For example, "arn%3Aaws%3Abedrock%3Aus-east-1%3A1234567890%3Aapplication-inference-profile%2Fabcdefghi", will override this.model in final /invoke URL call.
+      Must still provide `model` as normal modelId to benefit from all the metadata.
+  */
+  modelAlias?: string;
```

Comment:
Should we call this `applicationInferenceProfile`?

Reply:
Works for me. Figured to make it generic tho only application inference profile is the only use case today.

Reply:
@jacoblee93 friendly bump, hoping to stay w/ LangChain as we migrated to App Inf profiles. Gotta deliver this soon. Thanks!

Reply:
Will try to get this in today thanks!